rules.py

```diff
@@ -18,9 +18,12 @@
 import argparse
 import io
 import os.path
+import hashlib
+from datetime import datetime
 import pycountry
 import requests
 from lxml import etree
+from github import Github
```

Comment:
Pygithub is a new dependency and is not installed on your machine. Run

sudo pip install pygithub